static/index.js

```diff
@@ -22,19 +22,43 @@ function touchMove(ev) {
   const offsetX = ev.changedTouches[0].clientX - start.x;
   const offsetY = ev.changedTouches[0].clientY - start.y;
 
-  ws.send(JSON.stringify({ type: 'move', x: offsetX * 2, y: offsetY * 2 }));
+  ws.send(JSON.stringify({type: "move", x: offsetX * 2, y: offsetY * 2}));
 
   start.x = ev.changedTouches[0].clientX;
   start.y = ev.changedTouches[0].clientY;
 }
 
-document.addEventListener('DOMContentLoaded', (event) => {
-  document.getElementById('touchpad').addEventListener('touchstart', touchStart);
-  document.getElementById('touchpad').addEventListener('touchmove', touchMove);
+document.addEventListener("DOMContentLoaded", event => {
+  document
+    .getElementById("touchpad")
+    .addEventListener("touchstart", touchStart);
+  document.getElementById("touchpad").addEventListener("touchmove", touchMove);
+  var Tap = new Hammer.Tap({
+    event: "tap",
+    taps: 1
+  });
+  var DoubleTap = new Hammer.Tap({
+    event: "doubletap",
+    taps: 2
+  });
+  const hammer = new Hammer.Manager(document.getElementById("touchpad"));
+
+
+  var singleTap = new Hammer.Tap({ event: 'tap' });
+  var doubleTap = new Hammer.Tap({event: 'doubletap', taps: 2 });
 
-  const hammer = new Hammer(document.getElementById('touchpad'));
-  hammer.on('tap', (ev) => {
+  hammer.add([doubleTap, singleTap]);
+  doubleTap.recognizeWith(singleTap);
```

Comment:
Do we want this? https://hammerjs.github.io/recognize-with/

Reply:
BTW, you can use this for testing https://gutool.com/test-mouse/

Reply:
I think if ever there's a need to add a middle mouse click then this way would make it straightforawrd

Reply:
Or perhaps, scrolling (two finger swipe?)

Reply:
Yes, I think scrolling will be useful, I will create an issue for the same.